<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdgeToolDialog</class>
 <widget class="QDialog" name="EdgeToolDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edge Tool</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QListWidget" name="edgeList"/>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Dash Length:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="dashLen"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Dash Gap:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="dashGap"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
